amplify.yml

```diff
@@ -6,7 +6,7 @@ frontend:
         - npm ci
     build:
       commands:
-        - env | grep -e NEXTAUTH_URL -e NEXTAUTH_SECRET -e ZITADEL_CLIENT_ID -e ZITADEL_ISSUER -e MUMBLE_API_URL -e ENVIRONMENT  >> .env
+        - env | grep -e NEXTAUTH_URL -e NEXTAUTH_SECRET -e ZITADEL_CLIENT_ID -e ZITADEL_ISSUER -e MUMBLE_API_URL -e ENVIRONMENT -e TEST_USER_NAME -e TEST_USER_PASSWORD >> .env
```

Comment:
Do we need to set the test user credentials on the amplify? 
We are only building on merge. (see amplify.yml).  if we are going to run the tests on release, then we would need to change the config of the amplify to something like npm run test && npm run build. I would not test anymore while releasing. What do you think?